.gitbook/FAQ.md

```diff
@@ -163,7 +163,7 @@ Technically, an L1 token can have multiple representations on L2. This mean that
 
 Yes! Please refer to [Boba Faucets](contents/developer/faucets.md).
 
-For more information on Boba’s testnet and fountain, [check out our documentation](https://docs.boba.network/developer-docs/multichain).
+For more information on Boba’s testnet and fountain, [check out our documentation](https://docs.boba.network/developer/multichain).
```

Comment:
Outside the scope of this PR, but the link doesn't seem relevant -- only lists mainnet addresses.

Reply:
Updated to the relative link